demos/monge_ampere1.py

```diff
@@ -91,7 +93,7 @@ def ring_monitor(mesh):
 
 
 # With an initial mesh and a monitor function, we are able to construct a
-# :class:`~.MongeAmpereMover` instance and adapt the mesh. By default, the Monge-Ampère
+# :class:`~movement.monge_ampere.MongeAmpereMover` instance and adapt the mesh. By default, the Monge-Ampère
```

Comment:
I'm not sure why this wasn't working originally. In laplacian_smoothing.py there is :class:`~.LaplacianSmoother` which works as it should, but it doesn't in other demos :)